onnxruntime/core/optimizer/selection_and_optimization_func.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_optimizer_registry.h"
+#include "core/framework/compute_capability.h"
+#include "core/graph/graph_viewer.h"
+
+namespace onnxruntime {
+static const std::string kConstantFoldingDQ = "ConstantFoldingDQ";
+
+/**
+ * Optimizer's selection function: Selects a set of nodes from a given graph for optimization. Additional key/value strings can be provided to configure the optimizer.
+ *                                 If needed, use graph_optimizer_registry to access the session options, the CPU EP and the logger.
+ *
+ * Optimizer's optimization function: Gets the nodes in ComputeCapability from nodes_to_optimize. Use graph_optimizer_registry to access the session options, the CPU EP
+ *                                    and the logger if needed to create the optimizer. Run optimization on the nodes/subgraph, and finally, update the ComputeCapability.
+ *
+ */
```

Comment:
Can we put these in the same place we define the functions signatures? graph_optimizer_registry.h would be where I'd look first. We have `using SelectionFunc = std::function...` there already. Can we define an alias for the optimization function next to that so they're both in the same place?